etc/bench.py

```diff
@@ -0,0 +1,27 @@
+import itertools
+
+from trading_calendars.calendar_utils import (
+    _default_calendar_aliases,
+    _default_calendar_factories,
+    TradingCalendarDispatcher,
+)
+
+
+def construct_all_calendars():
+    dispatcher = TradingCalendarDispatcher(
+        calendars={},
+        calendar_factories=_default_calendar_factories,
+        aliases=_default_calendar_aliases,
+    )
+
+    calendar_names = itertools.chain(
+        _default_calendar_aliases, _default_calendar_factories
+    )
+
+    for name in calendar_names:
+        assert dispatcher.get_calendar(name) is not None
+        dispatcher.deregister_calendar(name)
+
+
+def test_calendar_construction(benchmark):
+    benchmark(construct_all_calendars)
```

Comment:
Should we benchmark just the loop above that does the construction (as opposed to the dispatcher setup, etc, as well)?

Reply:
I don't think the overhead of dict construction is significant relative to constructing all the calendars. It would be more accurate, but I don't think would give you a different answer.

Reply:
Sounds fine then.